docs/applications/containers/troubleshooting-kubernetes/index.md

```diff
@@ -120,21 +127,29 @@ Enter `exit` to later leave this shell.
 
 ### Viewing Master and Worker Logs
 
-The locations of logs on your master nodes are:
+If you have set up Kubernetes using a method that does not include [**systemd**](https://linode.com/docs/quick-answers/linux-essentials/what-is-systemd/), the location of logs on your master nodes are in most cases as follows:
```

Comment:
Gonna reword this to be a little simpler: "If your nodes run systemd". Also going to add H4s for `#### non-systemd` and `#### systemd`